src/settings/App.tsx

```diff
@@ -47,13 +50,50 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 		sortOrder: 1
 	};
 
+	constructor(props) {
+		super(props);
+		// Enable the app if the user is member of a group with delegated admin rights
+		this.api.isGroupFoldersAdmin(OC.getCurrentUser().uid).then((isAdmin) =>  {
```

Comment:
Should be using `@nextcloud/auth` which also comes with type definitions:

https://nextcloud.github.io/nextcloud-auth/modules/_index_.html#getcurrentuser

Reply:
not needed anymore since access control is fully done in the backend now